ui/src/shared/components/ConfirmButton.tsx

```diff
@@ -97,7 +90,7 @@ class ConfirmButton extends PureComponent<Props, State> {
     this.setState({expanded: false})
   }
 
-  private calculatePosition = () => {
+  private get calculatePosition() {
```

Comment:
what do you think about changing the name of this? this name makes me think its an action and not a getter. maybe `calculatedPosition`

Reply:
So semantic!